library/pk_wrap.c

```diff
@@ -817,17 +778,17 @@ const mbedtls_pk_info_t mbedtls_eckey_info = {
     "EC",
     eckey_get_bitlen,
     eckey_can_do,
-#if defined(MBEDTLS_ECDSA_C)
-    eckey_verify_wrap,
-    eckey_sign_wrap,
+#if defined(MBEDTLS_PK_CAN_ECDSA_SOME)
```

Comment:
On second thought, it doesn't seem quite right for both the verify and sign function to be guarded by the `_SOME` macro. Shouldn't the verify function be guarded by the `_VERIFY` macro and the sign function by the `_SIGN` macro?